javascript/javascript2/week2/index.js

```diff
@@ -0,0 +1,13 @@
+console.log("js2 week2")
+let numbers = [1, 2, 3, 4, 6, 7];
+let newNumbers = [];
+
+const doubledNumbers = numbers.filter((number)=>{
+    if(number%2==0){
+        console.log(`even numbers are: ${number}`)
+    }else if(number%2!==0){
+        let oddNumbers = number*2;
+        newNumbers.push(oddNumbers);
+    }
+});
```

Comment:
you can use the chaining property here using filter and map to print the odd numbers as doubled